src/shogun/base/AnyParameter.h

```diff
@@ -39,7 +39,8 @@ namespace shogun
 		GRADIENT = 1u << 1,
 		MODEL = 1u << 2,
 		AUTO = 1u << 10,
-		READONLY = 1u << 11
+		READONLY = 1u << 11,
+		RUNFUNCTION = 1u << 12
```

Comment:
we dont have a property for dynamic parameters (those that are registered with watch_method).
Should be unified or?

Reply:
this makes a distinction between const functions, i.e. can be run with get, and non-const function which have side effects, so have to be called with run

Reply:
Ah ok, maybe put some docstring that makes this clear?